tests/test_blockchain_api.py

```
@@ -0,0 +1,102 @@
+    id1 = response1.json()["id"]
+
+    # Get issue 1 details to get its hash
+    issue1 = db_session.query(Issue).filter(Issue.id == id1).first()
+    hash1 = issue1.integrity_hash
+    assert hash1 is not None
+    assert issue1.previous_integrity_hash == ""
+
+    # Verify issue 1
+    v_response1 = client.get(f"/api/issues/{id1}/blockchain-verify")
+    assert v_response1.status_code == 200
+    assert v_response1.json()["is_valid"] == True
```

Ruff (0.15.1)

[error] Avoid equality comparisons to `True`; use `v_response1.json()["is_valid"]:` for truth checks

Replace with `v_response1.json()["is_valid"]`

(E712)